react-native.config.js

```diff
@@ -0,0 +1,68 @@
+//
+// Copyright (c) Microsoft Corporation
+//
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+//
+// @ts-check
+
+/**
+ * @template ConfigT
+ * @typedef {{
+ *   name: string,
+ *   description?: string,
+ *   func: (argv: Array<string>, config: ConfigT, args: {}) => ?Promise<void>,
+ *   options?: Array<{
+ *     name: string,
+ *     description?: string,
+ *     parse?: (val: string) => any,
+ *     default?:
+ *       | string
+ *       | boolean
+ *       | number
+ *       | ((config: ConfigT) => string | boolean | number),
+ *   }>,
+ *   examples?: Array<{
+ *     desc: string,
+ *     cmd: string,
+ *   }>,
+ * }} Command;
+ */
+
+/** @type {{ commands: Command<{}>[] }} */
+module.exports = {
+  commands: [
+    {
+      name: "init-test-app",
+      description: "Initializes a new test app project",
+      func: (_, __, args) => {
+        const path = require("path");
+        const plop = require("node-plop");
+        // @ts-ignore tsc doesn't think that "node-plop" returns a function
+        plop(path.join(__dirname, "plopfile.js"), {
+          destBasePath: args["destination"],
+        })
+          .getGenerator("app")
+          .runActions(args);
+      },
+      options: [
+        {
+          name: "--destination [string]",
+          description: "Path to the destination directory",
```

Comment:
This is more a long-form version of the option name, so perhaps it can be rephrased to something like:

```suggestion
          description: "Path to the directory where the test app should be created",
```